src/main/java/ch/njol/skript/events/EvtRealTime.java

```diff
@@ -0,0 +1,146 @@
+				.description("Called when the local time of the system the server is running on reaches the provided real-life time.")
+				.examples(
+					"at 14:20 in real time:",
+					"at 2:30am real time:",
+					"at 6:10 pm in real time:",
+					"at 5:00 am and 5:00 pm in real time:",
+					"at 5:00 and 17:00 in real time:"
+				)
+				.since("INSERT VERSION");
+
+		TIMER = new Timer("EvtSystemTime-Tasks");
+	}
+
+	private Literal<Time> times;
+	private boolean unloaded = false;
+	private final List<RealTimeInfo> infoList = new ArrayList<>();
+
+	@Override
+	public boolean init(Literal<?>[] args, int matchedPattern, ParseResult parseResult) {
+		//noinspection unchecked
+		times = (Literal<Time>) args[0];
+		return true;
+	}
+
+	@Override
+	public boolean postLoad() {
+		Calendar currentCalendar = Calendar.getInstance();
+		currentCalendar.setTimeZone(TimeZone.getDefault());
+		for (Time time : times.getArray()) {
+			Calendar expectedCalendar = Calendar.getInstance();
+			expectedCalendar.setTimeZone(TimeZone.getDefault());
+			expectedCalendar.set(Calendar.MINUTE, time.getMinute());
+			expectedCalendar.set(Calendar.SECOND, 0);
+			expectedCalendar.set(Calendar.MILLISECOND, 0);
+			expectedCalendar.set(Calendar.HOUR_OF_DAY, time.getHour());
+			// Ensure the execution time is in the future and not the past
+			while (expectedCalendar.before(currentCalendar)) {
+				expectedCalendar.add(Calendar.HOUR_OF_DAY, 24);
+			}
+			RealTimeInfo info = new RealTimeInfo(time, expectedCalendar.getTimeInMillis());
+			infoList.add(info);
+			createNewTask(info);
+		}
+		return true;
+	}
+
+	@Override
+	public void unload() {
+		unloaded = true;
+		for (RealTimeInfo info : infoList) {
+			if (info.task != null)
+				info.task.cancel();
+		}
+		TIMER.purge();
+	}
+
+	@Override
+	public boolean check(Event event) {
+		throw new UnsupportedOperationException();
+	}
+
+	@Override
+	public boolean isEventPrioritySupported() {
+		return false;
+	}
+
+	@Override
+	public String toString(@Nullable Event event, boolean debug) {
+		return "at " + times.toString(event, debug) + " in real time";
+	}
+
+	private void execute() {
+		RealTimeEvent event = new RealTimeEvent();
+		SkriptEventHandler.logEventStart(event);
+		SkriptEventHandler.logTriggerStart(trigger);
+		trigger.execute(event);
+		SkriptEventHandler.logTriggerEnd(trigger);
+		SkriptEventHandler.logEventEnd();
+	}
+
+	private void preExecute(RealTimeInfo info) {
+		// Safety check, ensure this 'EvtServerTime' was not unloaded
+		if (unloaded)
+			return;
+		// Bump the next execution time by the appropriate amount
+		info.executionTime += HOUR_24_MILLISECONDS;
+		// Reschedule task for new executionTime
+		createNewTask(info);
+		// Activate trigger
+		execute();
+	}
+
+	private void createNewTask(RealTimeInfo info) {
+		TimerTask task = new TimerTask() {
+			@Override
+			public void run() {
+				preExecute(info);
+			}
+		};
+		info.task = task;
+		TIMER.schedule(task, new Date(info.executionTime));
+	}
```

Comment:
i'd prefer this to be up by check() since generally classes are organized as constructors -> functional methods -> getters/setters -> toString, but I don't think our conventions mandate anything about private helper methods.